scanner/src/routes/+page.svelte

```diff
@@ -3,6 +3,10 @@
   import { BarcodeScanner } from "@capacitor-community/barcode-scanner"
   import { Buffer } from "buffer"
   import { decode as decodeBase45 } from "base45"
+  import qrcode from '../assets/QRcode.svg';
+  import question from '../assets/question.svg';
+  import subtract from '../assets/Subtract.png';
```

Comment:
Could we please set up a path alias for tsconfig and svelte.config.js to make this into `$assets` instead? the `..` stuff starts getting ugly quick.

Also please rename all asset files to lowercase.